src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,64 @@
+package mate.academy.spring.model;
+
+import java.util.NoSuchElementException;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "roles")
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(value = EnumType.STRING)
+    private RoleName roleName;
+
+    public Role() {
+    }
+
+    public Role(RoleName roleName) {
+        this.roleName = roleName;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public RoleName getRoleName() {
+        return roleName;
+    }
+
+    public void setRoleName(RoleName roleName) {
+        this.roleName = roleName;
+    }
+
+    @Override
+    public String toString() {
+        return "Role{"
+               + "id=" + id
+               + ", roleName=" + roleName + '}';
+    }
+
+    public static Role.RoleName getName(String roleName) {

```

Comment:
```suggestion
    public static Role.RoleName getByName(String roleName) {
```
Also we can move this method inside of RoleName enum